plugins/headless/hu.bme.mit.gamma.headless.api/docs/headless-gamma-document.md

```diff
@@ -29,21 +29,19 @@ Additionally, cURL can be installed to test requests to the webserver, using the
 ## Step 2 - Importing projects
 
 
-Import the `hu.bme.mit.gamma.headless.api` and `hu.bme.mit.gamma.headless.source.generate` projects to your workspace, which already contains the necessary Gamma plugins that you want to export.
+Import the `hu.bme.mit.gamma.headless.api` project to your workspace, which already contains the necessary Gamma plugins that you want to export.
 
```

Comment:
Delete unnecessary new lines, only one should remain.

Reply:
Deleted unnecessary new lines.